src/sdk/resources/transferSecurityMappings.ts

```diff
@@ -0,0 +1,15 @@
+import { IHttpClient } from '../http';
+import { getPagedResponse, getTypedResponse, PagedResource, PagedResult } from './resource';
+import { TransferSecurityMapping} from '../models';
+
+export class TransferSecurityMappingsResource extends PagedResource<TransferSecurityMapping> {
```

Comment:
I would just derive from `BaseResource` since there is nothing editable and no other valid method besides `page` and that is re-implemented below.

Reply:
Looks like `IDataStoreIdentifier` accepts `Resource` instead of `BaseResource` (https://github.com/portalarchitects/skysync-vnext/blob/6be097feef342df6b63a8bfd7b66086b110901a1/Hosting/web/app/data/stores/store.ts#L65). It also looks like our client code relies on some of the functions and properties defined in `Resource`. I'm going to refactor again to use `Resource` unless you know of another way around those dependencies.